AIMS/Views/Home/AssetDetailsComponent.cshtml

```diff
@@ -1,141 +1,185 @@
-@await Html.PartialAsync("_AssetAddForm")  <!-- Calls the Add Asset Form-->
+@using System.Text.Json
+
+@* ---------- Top-level partials (modals/forms) ---------- *@
+@await Html.PartialAsync("_AssetAddForm")
 @await Html.PartialAsync("_EditAssetForm")
 @await Html.PartialAsync("_AddCategoryForm")
 
+@* ---------- Per-view styles (make sure _Layout.cshtml has @RenderSection("Styles", required: false) inside <head>) ---------- *@
+@section Styles {
+  <link rel="stylesheet" href="~/css/AssetDetailsComponent.css" asp-append-version="true" />
+  <link rel="stylesheet" href="~/css/_AssetButton.css"          asp-append-version="true" />
+  <link rel="stylesheet" href="~/css/_AddAssetForm.css"         asp-append-version="true" />
+  <style>
+    /* Small toolbar for the Manage Asset button */
+    .asset-toolbar{
```

Comment:
Is there a reason why you chose to embed this CSS in this file vs creating a new CSS file?